src/state/psarstake/hederaHooks.ts

```diff
@@ -196,6 +208,191 @@ export function useDerivativeHederaSarStake(positionId?: BigNumber) {
   );
 }
 
+/**
+ * This hook return the rent value of a position
+ * @param positionId The id of position
+ * Returns rent value in tiny bars
+ */
+function useHederaRent(positionId: string | undefined) {
```

Comment:
looks like this hook is only useful for unstake. lets rename this to make it more meaningful for specific position, otherwise this will be considered as global hook to find rent in hedera which is not correct

Reply:
This will be used in claim and compound too